python/cugraph/cugraph/datasets/metadata/dining_prefs.yaml

```diff
@@ -0,0 +1,23 @@
+name: dining_prefs
+file_type: .csv
+author: J.L. Moreno
+url:  https://data.rapids.ai/cugraph/datasets/dining_prefs.csv
```

Comment:
All the metadata fields are standardized now so the `url` field is at the very end.